new/LIT7100QeneLeyon.xml

```diff
@@ -0,0 +1,86 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7100QeneLeyon" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ልዮን፡ ወአውጣኪ፡ ምስለ፡ ድዮስቆሮስ፡ ሊቅ፡ በባሕለ፡ እሙን፡ ኢሐብሩ።</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Lǝyon wa-ʾAwṭāki mǝsla Dǝyosqoros liq ba-bāḥla ʾǝmun ʾi-ḥabru (Qǝne of the wāzemā-type)</title>
+                <title xml:lang="en" corresp="#t1">Leo and Eutyches did not agree with the master Dioscorus in the utterance of truth…</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="JK"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="UppQ1524o"/>
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p><foreign xml:lang="gez">Wāzemā</foreign>-type <foreign xml:lang="gez">qǝne</foreign> 
+                    attributed in the manuscript to a "Twoldo" who cannot be further identified.
```

Comment:
I think we add our interpretation of "Twoldo" as "Tawalda", which is quite secure, I think.

Reply:
Well, I think creating an ID for "Twoldo"/"Tawalda" him would not be wrong.

Reply:
Ok, I will let create it